### PR TITLE
add elevation group to collateral

### PR DESCRIPTION
**Pull request by lukaspetrasek**

### CODE REVIEW

File: src/loans/kamino.py [hunk 1]
```diff
@@ -280,14 +302,18 @@ def save_health_ratios(self) -> None:
...
+                    self.elevation_groups_to_liquidation_threshold
+                )
                 new_health_ratio = KaminoHealthRatio(
                     slot=int(self.last_slot),
                     user=loan_entity.obligation,
                     health_factor=loan_entity.health_ratio(),
                     std_health_factor=loan_entity.std_health_ratio(),
                     collateral=str(round(loan_entity.collateral_market_value(), 6)),
-                    risk_adjusted_collateral=str(round(loan_entity.collateral_market_value(), 6)),
+                    risk_adjusted_collateral=str(round(loan_entity.collateral_risk_adjusted_market_value(), 6)),
```
Comment (nickgore) [inline src/loans/kamino.py:316]: did nt we want to keep it unadjusted?

File: src/loans/kamino.py [hunk 2]
```diff
@@ -189,8 +200,8 @@ def std_health_ratio(self) -> str:
             return 'inf'
         if self.is_zero_deposit:
             return '0'
-        deposited_value = self.collateral_market_value()
-        borrowed_value = self.debt_risk_adjusted_market_value()
+        deposited_value = self.collateral_risk_adjusted_market_value()
+        borrowed_value = self.debt_market_value()
```
Comment (nickgore) [inline src/loans/kamino.py:204]: not sure if its correct. do we need to adjust for borrow factor for loans without special elevation group?

Comment (lukaspetrasek) [reply src/loans/kamino.py:204]: That's what I want to test experimentally.

File: src/loans/kamino.py (hunk 1 above)
Comment (lukaspetrasek) [reply src/loans/kamino.py:316]: That was before we found out how it's really done 😅

File: src/loans/kamino.py [hunk 3]
```diff
@@ -313,6 +339,18 @@ def get_price_for(self, token: str):
         assert price, f'failed to fetch price for {token}'
         return int(price) / SF
 
+    @staticmethod
+    def get_account_info(market: str, client: Client):
+        """ Fetch Solend obligations for pool. """
```
Comment (lukaspetrasek) [inline src/loans/kamino.py:344]:
```suggestion
        """ Fetch Kamino obligations for pool. """
```

File: src/loans/solend.py [hunk 4]
```diff
@@ -232,6 +232,7 @@ def set_initial_loan_states(self, initial_loan_states: pd.DataFrame) -> None:
                         reserve=collateral_info['reserve'],
                         mint=mint,
                         amount=collateral_info['amount'],
+                        elevation_group=collateral_info['elevation_group']
```
Comment (lukaspetrasek) [inline src/loans/solend.py:235]: Won't this break Solend?

Comment (nickgore) [reply src/loans/solend.py:235]: it definitely will. i ll backlog it.

**Pull request merged**